### PR TITLE
Do not require X11 to be available

### LINKED ISSUE #217: Does gnome-abrt work in Wayland?
There is a piece of code in gnome-abrt which checks if an Xserver is available and quits immediately if it is not. Now I wonder: what if we have Wayland and there is no Xserver deliberately? Do we really need to check if an Xserver is available? Is there any reason why gnome-abrt should not work in Wayland? Aren't the default error handling routines sufficient if a user runs gnome-abrt in non-graphic environment? Can we just drop this piece of code?

The checking code is in ```src/gnome_abrt/wrappers/xserver_check.c```.

### PR DESCRIPTION
**Pull request by rluzynski**
Nowadays we have Wayland and X11 is available only as a backward
compatibility feature.  Sooner or later it will be retired.

This commit also makes libX11 no longer explicitly required by
gnome-abrt.

This commit partially reverts commits:
* 5dcf9097cdfa68087569e13f9e8c2eb2eba8250f,
* 117465376548618eb3b3f6b69d5ae6a1ab95deec,
* 798d510734e970d3d54be64a4367fa2ae6de66f8.

Closes #217

### CODE REVIEW

File: src/gnome-abrt [hunk 1]
```diff
@@ -23,12 +23,7 @@ import sys
 import logging
 import signal
 import locale
-from gnome_abrt.wrappers import (show_events_list_dialog,
-                                 can_connect_to_xserver)
-
-if not can_connect_to_xserver():
-    sys.stderr.write("Cannot connect to X server\n")
-    sys.exit(1)
+from gnome_abrt.wrappers import (show_events_list_dialog)
```
Comment (ernestask) [inline src/gnome-abrt]: We can get rid of the parentheses here.

Comment (ernestask) [reply src/gnome-abrt]:
```suggestion
from gnome_abrt.wrappers import show_events_list_dialog
```

**Comment by ernestask** [comment]
Otherwise looks great, thanks!

File: src/gnome-abrt (hunk 1 above)
Comment (rluzynski) [reply src/gnome-abrt]: Thanks for the suggestion, I have applied it and updated this pull request.

**Comment by rluzynski** [comment]
Just a friendly ping to remind that the PR has been updated according to your suggestion.

**Comment by ernestask** [comment]
I haven’t forgotten, will take care of it today or tomorrow.

**Pull request merged**